aspnetcore/fundamentals/metapackage.md

```diff
@@ -32,13 +31,31 @@ You can use the package trimming process to remove packages that you don't use.
 
 The following *.csproj* file references the `Microsoft.AspNetCore.All` metapackage for ASP.NET Core:
 
-[!code-xml[](metapackage/samples/Metapackage.All.Example.csproj?highlight=6)]
+[!code-xml[](metapackage/samples/Metapackage.All.Example.csproj?highlight=8)]
+
+::: moniker range=">= aspnetcore-2.1"
+
+## Implicit versioning
+
+In ASP.NET Core 2.x, you can specify the `Microsoft.AspNetCore.All` package reference without a version. When the version isn't specified, an implicit version is specified by the SDK (`Microsoft.NET.Sdk.Web`). We recommend relying on the implicit version specified by the SDK and not explicitly setting the version number on the package reference. If you have questions about this approach, leave a GitHub comment at the [Discussion for the Microsoft.AspNetCore.App implicit version](https://github.com/aspnet/Docs/issues/6430).
```

Comment:
2.x => 2.1 and higher. I know it's guarded by the moniker range but seeing 2.x is still rather confusing.

Reply:
I was slightly concerned about saying "or later" in the context of 3.0 and our not changing this (by accident) when 3.0 arrives and perhaps `All` becomes unsupported. I think I can't put 3.0 on the moniker range at this time because the version probably hasn't been added yet to the build system ... it will probably :boom: if I do that. I'll say "or later" here, but brace yourself ... we don't have a stellar track record when we don't harden something on the spot for future changes. 🙈